test/k8sT/fqdn.go

```diff
@@ -85,6 +85,11 @@ var _ = SkipDescribeIf(helpers.RunsOn54Kernel, "K8sFQDNTest", func() {
 		err = kubectl.WaitforPods(helpers.DefaultNamespace, "-l zgroup=testapp", helpers.HelperTimeout)
 		Expect(err).Should(BeNil(), "Testapp is not ready after timeout")
 
+		// Validate that coredns is reachable from test pods
+		err = kubectl.NslookupInPod(helpers.DefaultNamespace, appPods[helpers.App2],
+			fmt.Sprintf("kube-dns.%s.svc.cluster.local", helpers.CiliumNamespace))
```

Comment:
I believe `kube-dns` always runs in `kube-system`, is it not true?

Reply:
You must be right, will fix :-)